eZ/Publish/API/Repository/Values/Content/Query/Criterion/FullText.php

```diff
@@ -21,15 +21,26 @@
  *
  * Normalization and querying capabilities might depend on the system
  * configuration or the used search engine and might differ. The following
- * basic query semantics are supported:
- *
- * - If multiple words are specified an AND query is performed. OR queries are
- *   not yet supported.
+ * query semantics are supported with Solr search engine:
```

Comment:
Last nitpick:

Could you change this to reflect the capability PR so we refer to a advanced fulltext cable engines and a basic fulltext cable engines here? to avoid hardcoding reference to engine names. Also fell free to refer the the method (`SearchService::supports()`), if we change that I'll adapt it.

Reply:
Updated in https://github.com/ezsystems/ezpublish-kernel/pull/2101/commits/d7a8dbb30e3f8bef7c9650000a0362aa9b56e371